index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="./index.css" />
+		<link
+			rel="stylesheet"
+			href="https://use.fontawesome.com/releases/v6.0.0/css/all.css"
+			integrity="sha384-3B6NwesSXE7YJlcLI9RpRqGf2p/EgVH8BgoKTaUrmKNDkHPStTQ3EyoYjCGXaOTS"
+			crossorigin="anonymous"
+		/>
+		<link rel="icon" href="./assets/images/react-logo.png" type="image/png" />
+		<title>React summit</title>
+	</head>
+
+	<body>
+		<div class="intro-section">
+			<header class="nav-header">
+				<nav class="nav-wrapper">
+					<a href="index.html"
+						><img src="./assets/images/logo.png" alt="logo" class="nav-logo"
+					/></a>
+					<i class="fa-solid fa-bars" id="menuBarIcon"></i>
+					<ul class="nav-list-group" id="nav-list-group">
+						<li id="closeMenu"><i class="fa-solid fa-xmark" id="closeMenuIcon"></i></li>
+						<li class="nav-list"><a href="./about.html" class="nav-list-link">About</a></li>
+						<li class="nav-list"><a href="#" class="page-links nav-list-link">Program</a></li>
+						<li class="nav-list"><a href="#" class="page-links nav-list-link">Join</a></li>
+						<li class="nav-list"><a href="#" class="page-links nav-list-link">Sponsor</a></li>
+						<li class="nav-list"><a href="#" class="page-links nav-list-link">News</a></li>
+						<li class="nav-list" id="nav-list">
+							<button type="button" class="page-links nav-list-button">CC campain</button>
+						</li>
+					</ul>
+				</nav>
+			</header>
```

Comment:
- [ ] While you've done a good job with the navigation, the dark bar is not present. Please take a look at the images shown below and implement it 👇 
#### Your Design
![image](https://user-images.githubusercontent.com/46839548/163293658-ea0abdca-89b9-4129-b1a3-d64d307f7aec.png)
#### Behance Design
![image](https://user-images.githubusercontent.com/46839548/163293778-938f0b9c-c40d-4ff6-903d-1a4bcb983954.png)